### PR TITLE
Resize the iTwin Logos in README's

### PR DESCRIPTION
**Pull request by AdamMeza-Bentley**
## Changes

- resizes the logo images in README

## Testing

N/A

## Docs

N/A

### CODE REVIEW

**Review by rohan-kadkol — COMMENTED** [review]
Noticed that your PR description had a stray bullet point. Not sure if you intended to add something more.

File: packages/itwinui-react/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <img
       src='https://itwinplatformcdn.azureedge.net/iTwinUI/iTwinUI-new-logo.png'
       alt='iTwinUI logo'
+      height='150'
```
[the remark this answers is not in the document]
Comment (AdamMeza-Bentley) [reply packages/itwinui-react/README.md]: that did work! and made the height attribute unnecessary

File: packages/itwinui-css/README.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
   <picture>
     <source
       media='(prefers-color-scheme: dark)'
-      srcset='https://itwinplatformcdn.azureedge.net/iTwinUI/iTwinUI-new-dark-logo.png'
+      srcset='https://itwin.github.io/iTwinUI/logo.svg'
```
Comment (rohan-kadkol) [inline packages/itwinui-css/README.md:5]: I think this should be the dark variant.

```suggestion
      srcset='https://itwin.github.io/iTwinUI/logo-dark.svg' 
```

This fixes the logo on the dark theme:

|Old|New|
|-|-|
|image|image|

**Review by rohan-kadkol — APPROVED** [review]

File: packages/itwinui-css/README.md (hunk 2 above)
Comment (AdamMeza-Bentley) [reply packages/itwinui-css/README.md]: made that change!

**Pull request merged**